### PR TITLE
Fix invalid Pod directives (issue #52).

### PR DESCRIPTION
**Pull request by shlomif**
See https://github.com/preaction/Log-Any/issues/52 . This is part of the
CPAN pull-req challenge.

### CODE REVIEW

File: README.mkdn [hunk 1]
```diff
@@ -4,7 +4,7 @@ Log::Any - Bringing loggers and listeners together
 
 # VERSION
 
-version 1.703
+version 1.704
```
Comment (preaction) [inline README.mkdn]: This is handled automatically by Dist-Zilla, so we don't need to update it manually.

**Comment by preaction** [comment]
Thanks! If you could just remove the patch to the README, I can merge this.

**Comment by shlomif** [comment]
On Sun, 03 Dec 2017 05:54:44 +0000 (UTC)
Doug Bell <notifications@github.com> wrote:

> Thanks! If you could just remove the patch to the README, I can merge this.
> 

Done! I should note that the README change happened after I ran "dzil test
--all" and I thought it was intentional.

-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
Stop Using MSIE - http://www.shlomifish.org/no-ie/

Friends may come and go, but enemies accumulate.
    — Thomas Jones (1892-1969)

Please reply to list if it's a mailing list post - http://shlom.in/reply .

**Comment by shlomif** [comment]
On Mon, 04 Dec 2017 19:05:32 +0000 (UTC)
Doug Bell <notifications@github.com> wrote:

> Merged #67.
> 

Thanks! :)

-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
Escape from GNU Autohell - http://www.shlomifish.org/open-source/anti/autohell/

Chuck Norris does not own a dishwasher. His dishes know better than to become
dirty.
    — http://www.shlomifish.org/humour/bits/facts/Chuck-Norris/

Please reply to list if it's a mailing list post - http://shlom.in/reply .

**Pull request merged**